monorepo/src/test/kotlin/com/freeletics/gradle/tasks/ComputeVersionCodeTest.kt

```diff
@@ -9,47 +9,30 @@ import org.junit.Test
 internal class ComputeVersionCodeTest {
 
     @Test
-    fun `when on a matching release branch it returns version computed from it`() {
-        val git = FakeGit(
-            branch = "release/fl/1.2.0",
-            describe = "fl/v1.0.0",
-            commitsSince = 0,
-        )
-
-        assertThat(computeVersionCode(git, "fl", LocalDate.now())).isEqualTo(1020000)
-    }
-
-    @Test
-    fun `when on a matching release branch with extra commits it returns version computed from it`() {
+    fun `when there is a matching tag it returns version computed from it`() {
         val git = FakeGit(
-            branch = "release/fl/1.2.0",
-            describe = "fl/v1.0.0",
-            commitsSince = 23,
+            branch = "main",
+            describe = "fl/v4.3.1",
         )
 
-        assertThat(computeVersionCode(git, "fl", LocalDate.now())).isEqualTo(1020023)
+        assertThat(computeVersionCode(git, "fl", LocalDate.now())).isEqualTo(4030001)
     }
 
     @Test
-    fun `when on a matching release branch with too many extra commits it fails`() {
+    fun `when there is a matching tag with extra commits it returns version computed from it`() {
```

Comment:
This is practically the same test now as the one above it

Reply:
I left it at first but after I cleaned up more code it became like this. Removed it [024d557](https://github.com/freeletics/freeletics-gradle-plugins/pull/153/commits/024d557256066a7adafa17b2a4f4d497f9837b32)